### PR TITLE
Update solution files under src/libraries

### PR DESCRIPTION
**Pull request by ViktorHofer**
- [Update slngen version to work with .NET 7 SDKs](https://github.com/dotnet/runtime/commit/71a3985b40c05087579ba41ddb3ad371a1233096)

- [Batch update all solution files under src/libs](https://github.com/dotnet/runtime/commit/14f15ff1d7ccc53be5f5cb1b25768f719564ecdb) ... by invoking the following command: "dotnet.cmd build
src/libraries/slngen.proj".

@carlossanlop this includes your desired change in the System.Formats.Tar solution file.

### CODE REVIEW

File: .config/dotnet-tools.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
       ]
     },
     "microsoft.visualstudio.slngen.tool": {
-      "version": "8.1.6",
+      "version": "8.5.17",
```
Comment (carlossanlop) [inline .config/dotnet-tools.json:24]: Where is this tool located? Or alternative question: Where did you get this number from?

File: src/libraries/System.Formats.Tar/System.Formats.Tar.sln [hunk 2]
```diff
@@ -60,6 +66,7 @@ Global
 		HideSolutionNode = FALSE
 	EndGlobalSection
 	GlobalSection(NestedProjects) = preSolution
+		{BE259E6E-B4F5-47DC-93D5-204297098A8C} = {6CF0D830-3EE9-44B1-B548-EA8750AD7B3E}
```
Comment (carlossanlop) [inline src/libraries/System.Formats.Tar/System.Formats.Tar.sln:69]: Nice! Thanks. This is what I wanted: Nest the newly added StreamConformanceTests project under the tests folder.

**Review by carlossanlop — APPROVED** [review]
LGTM, assuming CI doesn't complain.
I just left a non-blocking question, for my own education.

File: .config/dotnet-tools.json (hunk 1 above)
Comment (ViktorHofer) [reply .config/dotnet-tools.json:24]: [n](https://www.nuget.org/packages/Microsoft.VisualStudio.SlnGen.Tool)

File: src/libraries/System.Formats.Tar/System.Formats.Tar.sln (hunk 2 above)
Comment (ViktorHofer) [reply src/libraries/System.Formats.Tar/System.Formats.Tar.sln:69]: Yes, this is all auto-generated by the slngen tooling.

**Comment by ViktorHofer** [comment]
> LGTM, assuming CI doesn't complain.

Yeah, sln files aren't validated in CI at all.

**Pull request merged**